modules/Behaviour/behaviour_manage_edit.php

```diff
@@ -100,18 +97,42 @@
                 $form->addHiddenValue('address', "/modules/Behaviour/behaviour_manage_add.php");
                 $form->addRow()->addClass('hidden')->addHeading('Step 1', __('Step 1'));
 
+
+                //To show other students involved in the incident
+                if(!empty($values['gibbonMultiIncidentID'])) {
+
+                    $students = $container->get(BehaviourGateway::class)->selectMultipleStudentsOfOneIncident($values['gibbonMultiIncidentID'])->fetchAll();
+                }
+                $students = $resultList->fetchAll();  
```

Comment:
It looks like you've auto accepted the suggested change, but haven't updated the rest of the file in context. Be sure to check this out, as this code will now create an error.